### ISSUE TITLE
White text on a white background on Pika’s website

### ISSUE THREAD

**Issue by pepelsbey** [superhighfives/pika#86]
**Describe the bug**

The Pika text shows white text on white background until all JS is loaded.

It takes around 13s for the background to appear with Slow 3G network throttling in Chrome DevTools, and about 5s on Fast 3G.

**To Reproduce**

1. Go to https://superhighfives.com/pika
2. Wait for the page to load

**Expected behavior**

Pika text on a contrasting background

**Screenshots**

![image](https://user-images.githubusercontent.com/105274/191947502-38fce0ff-b4dc-4d30-b31b-1a01842b655c.png)

How it might have been with a small fix:

```
background-image: linear-gradient(to right, #c321be, #e799c8);
```

![image](https://user-images.githubusercontent.com/105274/191947741-9ad4dd3e-31ab-4f88-b8b7-bd20868adf8b.png)

**Comment by superhighfives** [comment]
Oooh, good catch—got a review app with a quick fix. Does this resolve the issue? https://superhighfiv-background-zy4ps8.herokuapp.com/pika

**Comment by pepelsbey** [comment]
Yes it looks like a good quick fix, though I’d use a gradient ;) Thanks!

**Comment by superhighfives** [comment]
Great, sorted. Thank you!
